tests/test_converter.py

```diff
@@ -1647,10 +1647,12 @@ def test_diff_meas_circuit(self):
         qc1.measure(1, 1)
 
         qtemp, qtemp1 = load(qc), load(qc1)
-        assert qtemp()[0] == qml.measure(0) and qtemp1()[0] == qml.measure(2)
+        assert qml.equal(qtemp()[0], qml.measure(0))
```

Comment:
`qml.equal` for `MeasurementValue` compares the stored `MidMeasureMP`s, and their equality depends on the `id` of the measurement. So you would have to instead compare the `wires`, `reset`, and `postselect` attributes manually.

Maybe also worth changing `qml.equal` wrt `MidMeasureMP` 😅 .

Reply:
😬